### PR TITLE
Add PortionData to Double Plants.

### PR DESCRIPTION
**Pull request by me4502**
This adds PortionData to DoublePlants.

I'm not entirely sure if this is necessary, but already did the changes so may as well create the PR, just incase it is required.

### CODE REVIEW

File: src/main/java/org/spongepowered/common/mixin/core/block/MixinBlockDoublePlant.java [hunk 1]
```diff
@@ -45,12 +49,13 @@
 
     @Override
     public ImmutableList<ImmutableDataManipulator<?, ?>> getManipulators(IBlockState blockState) {
-        return ImmutableList.<ImmutableDataManipulator<?, ?>>of(getDoublePlantTypeFor(blockState));
+        return ImmutableList.<ImmutableDataManipulator<?, ?>>of(getDoublePlantTypeFor(blockState),
+                getPortionData(blockState));
```
Comment (kashike) [inline src/main/java/org/spongepowered/common/mixin/core/block/MixinBlockDoublePlant.java]: This doesn't need to be on a new line.

**Review by kashike — CHANGES_REQUESTED** [review]

File: src/main/java/org/spongepowered/common/mixin/core/block/MixinBlockDoublePlant.java (hunk 1 above)
Comment (me4502) [reply src/main/java/org/spongepowered/common/mixin/core/block/MixinBlockDoublePlant.java]: Fixed.

**Review by gabizou — APPROVED** [review]
Looks good.

**Comment by me4502** [comment]
Thanks, I'll do my final testing - as I've written a fair bit of extra code to test with since I did my last test.

**Pull request merged**